src/renderer/component/publishForm/view.jsx

```diff
@@ -1,884 +1,574 @@
+                {" "}
+                <Button fakeLink label={__('Learn more')} href="https://lbry.io/faq/naming" />
+              </p>
+              <div className="card__content">
+                <FormRow>
+                  <FormField
+                    stretch
+                    prefix={`lbry://${
+                      (channel === 'anonymous' || channel === 'new') ? '' : `${channel}/`
+                    }`}
+                    type="text"
+                    name="content_name"
+                    placeholder="myname"
+                    value={name || prefillClaim.name || ""}
+                    onChange={event => this.handleNameChange(event.target.value)}
+                    error={nameError}
+                    helper={(
+                      <BidHelpText
+                        uri={uri}
+                        isResolvingUri={isResolvingUri}
+                        winningBidForClaimUri={winningBidForClaimUri}
+                        claimIsMine={!!myClaimForUri}
+                        onEditMyClaim={this.editExistingClaim} />
+                    )}
+                    />
+                </FormRow>
+              </div>
+              <div className={classnames("card__content", { "card--disabled": !name })}>
+                <FormField
+                  className="input--price-amount"
+                  type="number"
+                  name="content_bid"
+                  step="any"
+                  label={__('Deposit')}
+                  postfix="LBC"
+                  value={bid}
+                  error={bidError}
+                  min="0"
+                  disabled={!name}
+                  onChange={event => this.handleBidChange(event.target.value)}
+                  helper={__('This LBC remains yours and the deposit can be undone at any time.')}
+                  placeholder={winningBidForClaimUri ? winningBidForClaimUri + 10 : 10}
+                />
+              </div>
+            </section>
+
+            <section className="card card--section">
+              <FormRow>
+                <FormField
+                  type="checkbox"
+                  name="content_is_mature"
+                  postfix={__('Mature audiences only')}
+                  checked={nsfw}
+                  onChange={(event) => updatePublishForm({ nsfw: event.target.checked })}
+                />
+              </FormRow>
+
+              <FormRow padded>
+                <FormField
+                  label={__('Language')}
+                  type="select"
+                  name="content_language"
+                  value={language}
+                  onChange={event => updatePublishForm({ language: event.target.value })}
+                  >
+                  <option value="en">{__('English')}</option>
+                  <option value="zh">{__('Chinese')}</option>
+                  <option value="fr">{__('French')}</option>
+                  <option value="de">{__('German')}</option>
```

Comment:
Can we add more languages than this? (Possibly out of scope, file separately if so)